executable/installer/iexpress/packaged-as-an-iexpress-self-extracting-archive.yml

```diff
@@ -0,0 +1,18 @@
+rule:
+  meta:
+    name: packaged as an IExpress self-extracting archive
+    namespace: executable/installer/iexpress
```

Comment:
this mirrors `executable/installer/installshield`, good

https://github.com/fireeye/capa-rules/blob/1ff994f7916d66e39b4b5b8dbb310d0e0b051f7f/nursery/packaged-as-an-installshield-installer.yml#L4

Reply:
we're starting to see the compiler vs packager split a bit (e.g. gcc is definitely a compiler, exe4j is right now under "compiler", and iexpress is definitely not a compiler and "installer" makes sense). no changes recommended, just highlighting this sorta annoying question.